h2d/Tile.hx

```diff
@@ -182,7 +194,7 @@ class Tile {
 	}
 
 
-	public static function fromColor( color : Int, ?width = 1, ?height = 1, ?alpha = 1., ?allocPos : h3d.impl.AllocPos ) : Tile {
+	public static function fromColor( color : Int, ?width = 1., ?height = 1., ?alpha = 1., ?allocPos : h3d.impl.AllocPos ) : Tile {

```

Comment:
For backward compatibility, we need to keep width : Int and height : Int so that  fromColor(0xFF0000,0.5) - which was color + alpha does not get turned into color + width in a silent manner